changelog/wave/v1.16.4.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Wave v1.16.4
+date: 2024-12-30
+tags: [wave]
+---
+
+## Feature updates and improvements 
+
+* Add trace execution elapsed time logic by @pditommaso in [https://github.com/seqeralabs/wave/pull/497](https://github.com/seqeralabs/wave/pull/497)
+* Fix missing workspaceId in launch API by @pditommaso in [https://github.com/seqeralabs/wave/pull/780](https://github.com/seqeralabs/wave/pull/780)
+* Remove unneeded user async api by @pditommaso in [https://github.com/seqeralabs/wave/pull/781](https://github.com/seqeralabs/wave/pull/781)
+
+**Full changelog**: [https://github.com/seqeralabs/wave/compare/v1.16.3...v1.16.4](https://github.com/seqeralabs/wave/compare/v1.16.3...v1.16.4)
```

Comment:
Lot of entries are missing https://github.com/seqeralabs/wave/releases/tag/v1.16.4